src/scribe_data/utils.py

```diff
@@ -619,3 +622,55 @@ def list_languages_with_metadata_for_data_type(language_metadata=_languages):
 def camel_to_snake(name: str) -> str:
     """Convert camelCase to snake_case."""
     return re.sub(r"(?<!^)(?=[A-Z])", "_", name).lower()
+
+
+# MARK : Check Dump
+
+
+def check_lexeme_dump_prompt_download(output_dir):
+    existing_dumps = list(Path(output_dir).glob("*.json.bz2"))
+    if existing_dumps:
+        rprint("[bold yellow]Existing dump files found:[/bold yellow]")
+        for dump in existing_dumps:
+            rprint(f"  - {Path(output_dir)}/{dump.name}")
+
+        user_input = input(
+            "\nDo you want to\n - Delete existing dumps,\n - Skip download,\n - Use existing latest dump\n -Download (n)ew version?\n [d/s/u/n]: "
```

Comment:
Maybe I might just be overlooking something, but does the `(n)` option still need to be added here?